db.yml

```diff
@@ -1818,37 +1818,70 @@ modules:
         kernel: true
         nid: 0xE2C40624
         functions:
+          ksceKernelCancelCallback: 0xC040EC1C
           ksceKernelCancelMutex: 0x7204B846
+          ksceKernelChangeThreadPriority2: 0x63DAB420
```

Comment:
`sceKernelChangeThreadPriority` and `sceKernelChangeThreadPriority2` have different NIDs, while in this change they have the same one. Were those two functions merged?